media-sound/denemo/denemo-2.3.0-r1.ebuild

```diff
@@ -17,31 +17,25 @@ KEYWORDS="~amd64 ~x86"
 # --enable-mem(no) memory debugging: needs Electric fence (efence), which
 #		is not available in portage. See https://github.com/boundarydevices/efence
 # --enable-gtk-doc-pdf(no) doesn't work
-IUSE="alsa +aubio debug +evince jack +fluidsynth gtk-doc +gtk3 nls +portaudio
-	+portmidi +rubberband static test"
+IUSE="alsa +aubio debug jack +fluidsynth gtk-doc nls +portaudio +portmidi
+	+rubberband static test"
```

Comment:
shouldn't it be `static-libs` instead? what does it do exactly?

Reply:
Good catch. Looks like it's a no-op. Configure has an --enable-static option, where the comment says it's to build static libraries, but the app doesn't install any libraries. I guess, we should remove it.

Reply:
The binaries are still dynamically linked even with `USE=static`

Reply:
yeah, I say drop it. if no one asks for it, and a package has no usedeps, drop static-libs wherever you encounter it.

Reply:
alright